tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -88,16 +97,22 @@ spec:
         fi
 
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
```

Comment:
Didn't we have a ticket for this? If so maybe you can close it out as done with RELEASE-1176

Reply:
Good catch. I did recall having a conversation about making it mandatory everywhere, but I didn't recall having a Jira for this. Found it and added a comment that it will get done as part of this: https://issues.redhat.com/browse/RELEASE-922